pages/news/2021-4-16-chang_presentation.mdx

```diff
@@ -0,0 +1,15 @@
+---
+date: 2021-04-16
+title: Research Colloquium Presentation by Dawa Chang
+---
+
+# Knowledge Graphs for Innovation
+## Dawa Chang - ESR 13
+### WU (Vienna University of Economics and Business)
```

Comment:
Semantically, this does not make sense :wink: 
The `#` lines are headlines - there are not meant for being used for formatting or highlighting things.

In general, you may want to think about using a template or some guideline so that the posts use the same formatting. At the moment, each ESR creates the posts however they want. :smile: